pkg/webhook/common.go

```diff
@@ -70,6 +70,20 @@ func (h *webhookHandler) getConfig(ctx context.Context) (*v1alpha1.Config, error
 	return cfg, h.client.Get(ctx, keys.Config, cfg)
 }
 
+// validateGatekeeperResources returns whether an issue is user error (vs internal) and any errors
```

Comment:
Wrong comment?

Reply:
Done

Reply:
I'm still seeing a comment for `validateGatekeeperResources` here when it should be for `isGatekeeperResource` (also the contents of the comment don't match the behavior).

Reply:
Sorry, made the change, did not push it. Sould be good now.